cli/template/base/src/env.mjs

```diff
@@ -1,3 +1,5 @@
+// @ts-check
```

Comment:
Do we need ts-check in JS files, given that [we have checkJs set to true in tsconfig](https://github.com/t3-oss/create-t3-app/blob/next/tsconfig.json#L11)?

Reply:
Yeah, you are right. it was some issue with my VSCode earlier, thx for pointing that out!